Terasology/MetalRenegades
# Add ore generation to dynamic world generator

An integration of Terasology/OreGeneration in the dynamic world generator. Currently only contains a simple pocket ore definition for coal ore, but can be easily expanded to include more resources that the player can collect.

### To test
- Start a new game with the dynamic world generator.
- Use the railgun to fire into the ground, making a 'cave'.
- Eventually there should be a coal ore pocket similar to the one pictured below:

![Terasology-190610165506-1920x1080](https://user-images.githubusercontent.com/44126397/59221458-e46f3680-8ba1-11e9-8cae-5d669f17b3c8.png)
